modules/publicize/ui.php

```diff
@@ -52,234 +51,23 @@ function management_page() { ?>
-										</li>
-
-										<?php
-									endforeach;
-									?>
-								</ul>
-							<?php endif; ?>
-
-
-
-							<?php
-								$connections = $this->publicize->get_connections( $name );
-								if ( empty ( $connections ) ) { ?>
-									<a id="<?php echo esc_attr( $name ); ?>" class="publicize-add-connection button" href="<?php echo esc_url( $connect_url ); ?>" target="_top"><?php echo esc_html( __( 'Connect', 'jetpack' ) ); ?></a>
-								<?php } else { ?>
-									<a id="<?php echo esc_attr( $name ); ?>" class="publicize-add-connection button add-new" href="<?php echo esc_url( $connect_url ); ?>" target="_top"><?php echo esc_html( __( 'Add New', 'jetpack' ) ); ?></a>
-			  					<?php } ?>
-			  			</div>
-			  		</div>
-				<?php endforeach; ?>
-				</div>
-				<script>
-				(function($){
-					$('.pub-disconnect-button').on('click', function(e){ if ( confirm( '<?php echo esc_js( __( 'Are you sure you want to stop Publicizing posts to this connection?', 'jetpack' ) ); ?>' ) ) {
-								return true;
-							} else {
-							e.preventDefault();
-							return false;
-						}
-					})
-				})(jQuery);
-				</script>
-			</div>
-
-			<?php wp_nonce_field( "wpas_posts_{$_blog_id}", "_wpas_posts_{$_blog_id}_nonce" ); ?>
-			<input type="hidden" id="wpas_ajax_blog_id" name="wpas_ajax_blog_id" value="<?php echo $_blog_id; ?>" />
-		</form><?php
-
-	}
-
-	public static function global_checkbox( $service_name, $id ) {
-		global $publicize;
-		if ( current_user_can( $publicize->GLOBAL_CAP ) ) : ?>
-			<p>
-				<input id="globalize_<?php echo $service_name; ?>" type="checkbox" name="global" value="<?php echo wp_create_nonce( 'publicize-globalize-' . $id ) ?>" />
-				<label for="globalize_<?php echo $service_name; ?>"><?php _e( 'Make this connection available to all users of this blog?', 'jetpack' ); ?></label>
-			</p>
-		<?php endif;
-	}
-
-	function broken_connection( $service_name, $id ) { ?>
-		<div id="thickbox-content">
-			<div class='error'>
-				<p><?php printf( __( 'There was a problem connecting to %s. Please disconnect and try again.', 'jetpack' ), Publicize::get_service_label( $service_name ) ); ?></p>
-			</div>
-		</div><?php
-	}
-
-	public static function options_page_other( $service_name ) {
-		// Nonce check
-		check_admin_referer( "options_page_{$service_name}_" . $_REQUEST['connection'] );
-		?>
-		<div id="thickbox-content">
-			<?php
-			ob_start();
-			Publicize_UI::connected_notice( $service_name );
-			$update_notice = ob_get_clean();
-			if ( ! empty( $update_notice ) )
-				echo $update_notice;
+		<h3 id="publicize"><?php esc_html_e( 'Publicize', 'jetpack' ) ?></h3>
+		<p><?php esc_html_e( 'Connect social media services to automatically share new posts.', 'jetpack' ) ?></p>
+		<h4><?php
+			printf(
+				wp_kses(
+					__( "We've recently made some updates to Publicize. Please visit the <a href='%s'>WordPress.com sharing page</a> to manage your publicize connections or use the button below.", 'jetpack' ),
```

Comment:
I'm afraid "recently" will become outdated pretty soon. Is there a better phrase or can we be sure to update it in, say, six months to something else?